src/DataGrid.tsx

```diff
@@ -548,16 +548,20 @@ function DataGrid<R, SR, K extends Key>(
     if (!isCellEvent && !isRowEvent) return;
 
     const { keyCode } = event;
+    // event.key may differ by keyboard input language, so we use event.keyCode instead
+    // event.nativeEvent.code cannot be used either as it would break copy/paste for the DVORAK layout
+    const cKey = 67;
+    const vKey = 86;
+
+    if (isCtrlKeyHeldDown(event) && keyCode === cKey && window.getSelection()?.toString() !== '') {
```

Comment:
Do you mean checking `window.getSelection() !== null`? When a cell is focused without highlighted text, `window.getSelection()` returns a selection object, if we check for `null`, it will just return and the cell won't be copied.

Reply:
I meant 

```
const selection = window.getSelection();
if (isCtrlKeyHeldDown(event) && keyCode === cKey && selection !== null && selection.toString() !== '') {
```

`window.getSelection()?.toString()` would return `undefined` if `window.getSelection()` is `null` and the condition will be true, no?

Reply:
When `handleKeyDown` is triggered, `window.getSelection()` seems to always return a selection object 🤔